### PR TITLE
Fix/signup email validation

### PR DESCRIPTION
**Pull request by Laurin-W**
Addresses issue https://github.com/assemblee-virtuelle/activitypods/issues/40.

### CODE REVIEW

File: frontend/src/pages/LocalLoginPage/SignupForm.js [hunk 1]
```diff
@@ -92,7 +104,7 @@ const SignupForm = ({ redirectTo }) => {
                 format={(value) =>
                   value
                     ? createSlug(value, {
-                        lang: 'fr',
+                        lang: navigator.language.substring(0, 2),
```
Comment (srosset81) [inline frontend/src/pages/LocalLoginPage/SignupForm.js:107]: `navigator.language` is only supported by [96% of browsers](https://caniuse.com/?search=navigator.language) so there's a risk it makes the browser crashes.
You should add a check. And maybe default to the app language (`process.env.REACT_APP_LANG`) ?

**Review by srosset81 — CHANGES_REQUESTED** [review]
Looks good ! Just a minor change.

File: frontend/src/pages/LocalLoginPage/SignupForm.js (hunk 1 above)
Comment (srosset81) [reply frontend/src/pages/LocalLoginPage/SignupForm.js:107]: Actually the check should be on the `navigator` object itself (https://caniuse.com/mdn-api_navigator)

Comment (Laurin-W) [reply frontend/src/pages/LocalLoginPage/SignupForm.js:107]: Oh wow, good to know!
`process.env.REACT_APP_LANG` is only available server-side though, right?
Maybe `navigator?.language?.substring(0, 2) || 'fr'` is an easy compromise then?

Comment (srosset81) [reply frontend/src/pages/LocalLoginPage/SignupForm.js]: No `process.env.REACT_APP_LANG` is available on the front, it's used just a few lines above :-)

Comment (Laurin-W) [reply frontend/src/pages/LocalLoginPage/SignupForm.js]: my bad :D changed it now

**Review by srosset81 — APPROVED** [review]

**Pull request merged**